src/lightning_app/frontend/panel/panel_frontend.py

```diff
@@ -27,25 +27,40 @@ def has_panel_autoreload() -> bool:
 
 
 class PanelFrontend(Frontend):
-    """The PanelFrontend enables you to serve Panel code as a Frontend for your LightningFlow.
+    """The `PanelFrontend` enables you to serve Panel code as a Frontend for your LightningFlow.
 
-    To use this frontend, you must first install the `panel` package:
+    Reference: https://lightning.ai/lightning-docs/workflows/add_web_ui/panel/
+
+    Parameters
+    ----------
+    entry_point : The path to a .py or .ipynb file, or a pure function.
+        The file or function must contain your Panel code.
+        The function can optionally accept an `AppStateWatcher` argument.
+
+    Raises
+    ------
+    TypeError : Raised if the `entry_point`provided is a class method
+
+    Example
+    -------
+
```

Comment:
We use this other format (I think it's called Google style):
```suggestion
    Args:
        entry_point: The path to a .py or .ipynb file, or a pure function. The file or function must contain your Panel code.
            The function can optionally accept an `AppStateWatcher` argument.

    Raises:
        TypeError: Raised if the `entry_point`provided is a class method

    Example:

```